### ISSUE TITLE
How to predict image tokens in kosmos-1 training phase？

### ISSUE THREAD

**Issue by mathfinder** [microsoft/unilm#1045]
How to predict the image tokens in auto-regressing for Kosmos-1 training：
1. Does it use the original input image embedding as Groundtruth？
2. What is the negative term in softmax？
